### DISCUSSION TITLE
Reference Sheet for Concept Art

### DISCUSSION THREAD

**Discussion by Ecoabismo** [gamerevar/proyecto-comunitario-1-gd#63] [Arte]
### Utilizar como referencia también el Concept Art de [Unity](https://github.com/gamerevar/proyecto-comunitario-1-unity/discussions/10)
***
PDF con las referencias para poder arrancar con el concept art. Este documento sirve para llevar al mínimo la cantidad de concept de alternativas en paralelo. Se cierra desde acá la dirección general, para que las itterations no se nos vallan de las manos y hagamos un cuello de botella para el resto del equipo.

La idea es empezar con los soldados y sus armas  y el templo y sus guardianes.
El estilo del concept en sí no es tan importante, ya que va a pasar por una estilización que va a depender del estilo del arte del juego, que aún está sin definir.

[Referencaias Juego comunitario Godot.pdf](https://github.com/gamerevar/proyecto-comunitario-1-gd/files/5743561/Referencaias.Juego.comunitario.Godot.pdf)

![Referencaias Juego comunitario Godot_Mesa de trabajo 1](https://user-images.githubusercontent.com/40045009/103158061-7c3e7080-4798-11eb-9702-44d8915beeb0.jpg)
![Referencaias Juego comunitario Godot-02](https://user-images.githubusercontent.com/40045009/103158062-7cd70700-4798-11eb-9eec-355b0a996235.jpg)
![Referencaias Juego comunitario Godot-03](https://user-images.githubusercontent.com/40045009/103158063-7e083400-4798-11eb-9677-5f8d9ec4d824.jpg)
![Referencaias Juego comunitario Godot-04](https://user-images.githubusercontent.com/40045009/103158064-7f396100-4798-11eb-8486-57bd19db3072.jpg)
![Referencaias Juego comunitario Godot-05](https://user-images.githubusercontent.com/40045009/103158059-79438000-4798-11eb-9733-7432c85fe970.jpg)

***
**Editado por @gamerevar**: Cuando suban el trabajo vinculado a una _tarjeta_ en _Codecks_, por favor incluyan el _$id_ para facilitar su actualización.

**Comment by gamerevar** [comment]
# Post original de @Ecoabismo 
***
Bueno hoy avanzamos banda: Acá está la base de lo que armamos en la llamada:
Si arman los concepts como el golemcito en psd o incluso a mano, lo ideal es que tenga ya la perspectiva de idle. Por ejemplo uno puede hacer en pixel art un concept que haya armado otro etc.
La seguimos luego!

![BaseForCharacter](https://user-images.githubusercontent.com/40045009/103327001-8d55de80-4a31-11eb-866e-7b67267ac162.png)
![BaseGolem](https://user-images.githubusercontent.com/40045009/103327003-8dee7500-4a31-11eb-86ef-769446632edd.png)
![BaseWithCharacter](https://user-images.githubusercontent.com/40045009/103327004-8e870b80-4a31-11eb-96ef-93f2bc9a2c1f.png)
![Golem](https://user-images.githubusercontent.com/40045009/103327005-8e870b80-4a31-11eb-8fe8-f54461c200a2.png)
![imagen_2020-12-29_202640](https://user-images.githubusercontent.com/40045009/103327007-8f1fa200-4a31-11eb-8a43-ccf040cdc4d0.png)

**Comment by Ecoabismo** [comment]
Chicos, esta es una especie de pseudoguia de como diseñar armas, tecnología de forma que no haya que hacer diferentes iterations, etc. Y recuerden que todo se va a reducir a pixeles reducidos, por lo que poner detalles de más es hacer trabajo que no vas a mostrar.

![Weaponmaking4](https://user-images.githubusercontent.com/40045009/103413885-2da72280-4b5a-11eb-8857-43945c953f2c.jpg)
![Weaponmaking1](https://user-images.githubusercontent.com/40045009/103413886-2f70e600-4b5a-11eb-8651-8067864d1bb5.jpg)
![Weaponmaking2](https://user-images.githubusercontent.com/40045009/103413887-2f70e600-4b5a-11eb-8ff9-95670c1182bb.jpg)
![Weaponmaking3](https://user-images.githubusercontent.com/40045009/103413889-30097c80-4b5a-11eb-8635-fe85f4f19ebc.jpg)
